spyder/plugins/onlinehelp/widgets.py

```diff
@@ -314,11 +313,13 @@ def load_history(self, history):
         """
         self.url_combo.addItems(history)
 
-    def initialize(self):
+    def initialize(self, val=True):

```

Comment:
Since this is a slot now to `sig_toggle_view_changed`, please decorate it above with `Slot(bool)`. Also, please change `val` to `checked`, which makes more sense.

Reply:
Fixed